data/en/resources.json

```diff
@@ -1,84 +1,85 @@
-      "copyright": "(c) 2018 Microsoft Corporation. All rights reserved.",
-      "DscResources": "AdcsCertificationAuthority AdcsCertificationAuthoritySettings AdcsEnrollmentPolicyWebService AdcsOnlineResponder AdcsWebEnrollment AdcsOcspExtension AdscTemplate",
-      "downloadCount": "129135",
-      "packageSize": "79650",
-      "tags": "DesiredStateConfiguration DSC DSCResourceKit DSCResource PSModule",
+      "summary": "DSC resources for installing, uninstalling and configuring Certificate Services components in Windows Server.",
+      "Authors": "DSC Community",
+      "DscResources": "AdcsAuthorityInformationAccess AdcsCertificationAuthority AdcsCertificationAuthoritySettings AdcsEnrollmentPolicyWebService AdcsOnlineResponder AdcsWebEnrollment AdscTemplate",
+      "lastUpdated": "30/04/2022 20:16:06 -07:00",
+      "created": "20/06/2020 23:17:06 -07:00",
+      "NormalizedVersion": "5.0.0",
+      "updated": "2022-04-30T20:16:06Z",
```

Comment:
It seems to error on this line, the first one with yesterdays date `"2022-04-30T20:16:06Z",`